src/main/java/uk/gov/hmcts/ccd/RestTemplateConfiguration.java

```diff
@@ -108,6 +141,7 @@ private HttpClient getHttpClient(final int timeout) {
                                 .useSystemProperties()
                                 .setDefaultRequestConfig(config)
                                 .setConnectionManager(cm)
+                                .setRetryHandler(new DefaultHttpRequestRetryHandler(3, false))
```

Comment:
During performance testing Fatih and I spotted an issue (org.apache.http.NoHttpResponseException) with server dropping connections under certain load and a solution on spring forums was suggested to add retry handler and also increase http.client.validate.after.inactivity property to 100 from intial 0 to help detect connections that have become stale (half-closed) while kept inactive in the pool.